docs/src/content/docs/getting-started/learning.md

```diff
@@ -0,0 +1,177 @@
+---
+title: Learning
+description: Explaining basic concepts, primitives and tools 
+sidebar: 
+  order: 2
+---
+## Installation
+
+First of all we will use @reatom/framework that incorparates many functions you may need in development.
+
+```bash
+npm install --save @reatom/framework
+```
+
+## Atoms are storing data and dependencies
+
+Let's have trivial example that employs reatom's reactivity by implementing simple `a + b = c`:
+
+```typescript
+import { createCtx, atom } from "@reatom/framework"
+
+const ctx = createCtx()
```

Comment:
Move `ctx` creation bellow. It is important to focus on the context last of all. Add extra comment that `ctx` created only once.